_includes/breadcrumbs.html

```diff
@@ -0,0 +1,33 @@
+{% assign breadcrumb_home = 'Home' %}
+{% if page.collection == 'authors' %}
+  {% assign breadcrumb_home = 'Analysis' %}
+{% endif %}
+{% assign breadcrumb_past = null %}
+{% assign breadcrumb_current = page.title %}
+
+{% if page.layout == 'post' %}
+  {% assign breadcrumb_current = page.title %}
+
+  {% if page.collection == 'resources' or page.collection == 'workshops' %}
+    {% assign breadcrumb_home = page.type %}
+  {% else %}
+    {% assign breadcrumb_home_path = site.collections | where: "label", page.collection | first %}
+    {% assign breadcrumb_home = breadcrumb_home_path.title %}
+  {% endif %}
+
+  {% if page.collection == 'briefs' %}
+    {% assign breadcrumb_past = page.brief_type %}
+  {% else %}
```

Comment:
The else part of this statement isn't necessary since breadcrumb_past was already set to null above.

Reply:
You're right, deleted this!